src/Compilers/CSharp/Portable/Symbols/ObsoleteAttributeHelpers.cs

```diff
@@ -96,6 +143,14 @@ internal static DiagnosticInfo CreateObsoleteDiagnostic(Symbol symbol, BinderFla
                 return null;
             }
 
+            if (data.Kind == ObsoleteAttributeKind.Experimental)
```

Comment:
>if (data.Kind == ObsoleteAttributeKind.Experimental) [](start = 12, length = 52)

Should experimental APIs be subject of BinderFlags.SuppressObsoleteChecks flag? Do we have a test targeting this scenario?